vj4/ui/templates/partials/contest_sidebar.html

```diff
@@ -0,0 +1,67 @@
+{% import "components/user.html" as user with context %}
+{% import "components/contest.html" as contest with context %}
+<div class="section side">
+{% if page_name != 'contest_detail' %}
+  <a class="contest-sidebar__bg" href="{{ reverse_url('contest_detail', tid=tdoc['_id']) }}">
+    <div class="section__header">
+      <h1>{{ tdoc['title'] }}</h1>
+    </div>
+    <div class="section__body">
+      <!-- TODO -->
+      {% if False %}
+      <span class="icon icon-check"></span> Attended
+      {% elif True %}
+      <span class="icon icon-schedule"></span> Not Attended
+      {% endif %}
+    </div>
+  </a>
+{% endif %}
+  <div>
+    <ol class="menu">
+    {% if page_name != 'contest_detail' %}
+      <li class="menu__item"><a class="menu__link" href="{{ reverse_url('contest_detail', tid=tdoc['_id']) }}">
+        <span class="icon icon-award"></span> {{ _('View Contest') }}
+      </a></li>
+    {% endif %}
+    {% if handler.has_perm(vj4.model.builtin.PERM_ATTEND_CONTEST) %}
+    {% if not attended %}
+      <li class="menu__item">
+        <form method="POST">
+          <!-- TODO -->
```

Comment:
@twd2 attend button can be in every page now. submit to self is not approachable.